### PR TITLE
Update performance.md to reflect existence of `useNativeDriver` in An…

### PR DESCRIPTION
**Pull request by abi**
…imated API

### CODE REVIEW

**Comment by hramos** [comment]
Thanks! Maybe we can later expand this and link to the Animated API doc from here.

File: docs/Performance.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Similarly, you can implement `shouldComponentUpdate` and indicate the exact cond
 
 "Slow Navigator transitions" is the most common manifestation of this, but there are other times this can happen. Using InteractionManager can be a good approach, but if the user experience cost is too high to delay work during an animation, then you might want to consider LayoutAnimation.
 
-The Animated api currently calculates each keyframe on-demand on the JavaScript thread, while LayoutAnimation leverages Core Animation and is unaffected by JS thread and main thread frame drops.
+The Animated api currently calculates each keyframe on-demand on the JavaScript thread (unless you set `useNativeDriver: true`), while LayoutAnimation leverages Core Animation and is unaffected by JS thread and main thread frame drops.
```
Comment (piotrek1543) [inline docs/Performance.md]: I think that braces here are useless and you may remove them. 

Also could you add this as link to this blog post: 
https://facebook.github.io/react-native/blog/2017/02/14/using-native-driver-for-animated.html? 
as it clearly explains why 'unless you set `useNativeDriver: true`' :-)

**Comment by piotrek1543** [comment]
@hramos  in comment to this MR I added a link to blog post about using native driver in animated. Would useful for planned extension? Or link to specic part of Animated API woud be enough?

**Comment by abi** [comment]
Thanks for the feedback @piotrek1543

**Pull request closed without merging**